test/drivers/test_suites/syscall_exit_suite/open_by_handle_at_x.cpp

```diff
@@ -120,6 +118,28 @@ TEST(SyscallExit, open_by_handle_atX_success)
 {
 	auto evt_test = get_syscall_event_test(__NR_open_by_handle_at, EXIT_EVENT);
 
+	auto fo = file_opener(".", (O_RDWR | O_TMPFILE | O_DIRECTORY));
+
+	if(!fo.get_fd())
+	{
+		FAIL() << "Error opening current directory" << std::endl;
+	}
+	struct file_handle *fhp;
+	fhp = (struct file_handle *)malloc(sizeof(*fhp) + sizeof(fhp->handle_bytes));
+	if(fhp == NULL)
+	{
+		FAIL() << "Error in allocating the `struct file_handle` with malloc" << std::endl;
+	}
+	int mount_id;
+	fhp->handle_bytes = 0;
+	if(syscall(__NR_name_to_handle_at, AT_FDCWD, fo.get_pathname(), fhp, &mount_id, 0) != 0)
```

Comment:
Could we only check for eg: `ENOTSUP` (or whatever the errno would be?); this could potentially shadow some real errors.

Reply:
I'd prefer to check agains `EOPNOTSUPP` given that the man clearly states that the syscall uses that error number.